src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/CourtCaseRestClient.java

```diff
@@ -73,14 +64,9 @@
                 .then();
     }
 
-    @Override
-    public Mono<Void> postMatches(String courtCode, String caseNo, GroupedOffenderMatches offenderMatches) {
-        return legacyCourtCaseRestClient.postMatches(courtCode, caseNo, offenderMatches);
-    }
-
-    @Override
-    public Mono<Void> postOffenderMatches(String caseId, String defendantId, GroupedOffenderMatches offenderMatches) {
+    private Mono<Void> postDefendantMatches(String caseId, String defendantId, GroupedOffenderMatches offenderMatches) {
```

Comment:
Is there a reason for naming this `postDefendantMatches` from the previous `postOffenderMatches` ? It depends how you look at it, of course. They are offender matches in the sense that the entities in teh matches are offenders and not defendants but defendant matches in the sense that they are being matched to the defendant but given the fact that the parameter is named "`OffenderMatches`"  - it's now inconsistent. I'd favour keeping the existing

Reply:
Yeah fair point, will stick with Offender. Can't recall quite why I decided Defendant was better but I think you're right